internal/engine/upper_test.go

```diff
@@ -3407,6 +3407,23 @@ func (f *testFixture) Start(manifests []model.Manifest, watchFiles bool, initOpt
 	f.startWithInitManifests(nil, manifests, watchFiles, initOptions...)
 }
 
+func (f *testFixture) StartAndWaitForInitialBuilds(manifests []model.Manifest, initOptions ...initOption) {
```

Comment:
I really don't want to create a new Start method variant every time we have a new batch of tests with new defaults.

Can you change your tests to call the existing start method?

Reply:
This is less a new start method and more a wrapper -- notice that we DO call the existing start method, and just do some checks after it completes. do you have suggestions for how to rename this so it's clearer what's going on?

Reply:
i would just create a separate WaitForInitialBuilds method and call that in the top-level test cases

Reply:
sure